src/widgets/about-video/ui/about-video.module.scss

```diff
@@ -0,0 +1,50 @@
+.video-wrapper {
+  padding: 24px 67px 0;
+
+  .video-container {
+    position: relative;
+
+    overflow: hidden;
+
+    width: 100%;
+    max-height: 600px;
+    padding-top: 56.25%;
```

Comment:
Why not pixels?

Reply:
@Quiddlee, looks like that's the reason https://hidde.blog/embedded-video-in-responsive-lay-outs/

Reply:
![image](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExeHhwaDRtZHh5ZjVxbTMzM2J1b2RodGZ1NHk2bzUydGlhYjZiMGZ0eCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/L3X9GvVhP1nY23Ah6u/giphy.webp)